### DISCUSSION TITLE
Soporte para placa iCEblink40-LP1K

### DISCUSSION THREAD

**Discussion by NintendoPlayer** [FPGAwars/icestudio#841] [New Boards Requests]
Buenas, acabo de adquirir la placa indicada en el título.  No se si está previsto darle soporte a esta placa, o si es algo muy costoso.  Preferí esta placa por el número de I/O que tiene.

Gracias por vuestra ayuda.

**Comment by javiverahacker** [comment]
buenas tardes tengo la placa iCEblink40-LP1K pero no me la reconoce el icestudio que puedo hacer porfavor, previamente intale el zadig

**Comment by javiverahacker** [comment]
tengo windows 10 instalado

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
